src/packet.rs

```diff
@@ -83,6 +88,20 @@ impl MspParser {
 		self.state == MspParserState::Header1
 	}
 
+	pub fn crc8_dvb_s2(crc: u8, a: u8) -> u8 {
```

Comment:
Why reinvent the wheel? You can simply use the `crc-any` crate, it is `no_std` compatible as well. This configuration should do fine:

```
let mut crc = CRC::create_crc(0xd5, 8, 0x0, 0x0, false);
crc.digest(&buf[..]);
crc.get_crc().try_into().unwrap()
```

Reply:
yes, you are right. refactored to use crc-any